From c50b30fee23a5d2fb8663e8828d027c6d7b25661 Mon Sep 17 00:00:00 2001
From: Li Xin <lixin117@meituan.com>
Date: Mon, 16 Sep 2019 11:57:46 +0800
Subject: [PATCH] robbyrussell: add hostname and time

---
 themes/robbyrussell.zsh-theme | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/themes/robbyrussell.zsh-theme b/themes/robbyrussell.zsh-theme
index 2fd5f2cd..af36b925 100644
--- a/themes/robbyrussell.zsh-theme
+++ b/themes/robbyrussell.zsh-theme
@@ -1,5 +1,6 @@
-PROMPT="%(?:%{$fg_bold[green]%}➜ :%{$fg_bold[red]%}➜ )"
+PROMPT="%m %(?:%{$fg_bold[green]%}➜ :%{$fg_bold[red]%}➜ )"
 PROMPT+=' %{$fg[cyan]%}%c%{$reset_color%} $(git_prompt_info)'
+RPROMPT="[%D{%y/%m/%f}|%*]"

 ZSH_THEME_GIT_PROMPT_PREFIX="%{$fg_bold[blue]%}git:(%{$fg[red]%}"
 ZSH_THEME_GIT_PROMPT_SUFFIX="%{$reset_color%} "
--
2.32.0
